internal/cluster/validations/validations.go

```diff
@@ -505,6 +505,16 @@ func ValidateClusterUpdateVIPAddresses(ipV6Supported bool, cluster *common.Clust
 
 	targetConfiguration.UserManagedNetworking = swag.Bool(false)
 	if params.UserManagedNetworking != nil {
+		if swag.BoolValue(params.UserManagedNetworking) {
+			err = ValidateVIPsWereNotSetUserManagedNetworking(cluster.APIVip, cluster.IngressVip,
+				cluster.APIVips, cluster.IngressVips, swag.BoolValue(cluster.VipDhcpAllocation))
+			if err != nil {
+				// reformat error to match order of actions
+				errParts := strings.Split(err.Error(), " cannot be set with ")
```

Comment:
I would write the error again for the reason that on debugging we tend to search for the error message to find the failure in the code.

Reply:
This will duplicate a lot of code.
How does this any different from any other string formatting?